platforms/hyperledger-indy/images/indy-cli/indy-ledger.sh

```diff
@@ -1,54 +1,28 @@
 #!/bin/bash
 
-vault_addr=$1
-vault_token=$2
-admin_path=$3
-admin_name=$4
-identity_path=$5
-identity_name=$6
-identity_role=$7
-pool_genesis_path=$8
-
-[ -z "$vault_addr" ] && { echo "Script Failed: Vault Address Missing"; exit 1;} || echo "Vault Address Received";
-[ -z "$vault_token" ] && { echo "Script Failed: Vault Token Missing"; exit 1;} || echo "VAult Token Received";
-[ -z "$admin_path" ] && { echo "Script Failed: Admin DID Path Missing"; exit 1;} || echo "Admin DID Path Received";
-[ -z "$admin_name" ] && { echo "Script Failed: Admin Name Missing"; exit 1;} || echo "Admin Name Received";
-[ -z "$identity_path" ] && { echo "Script Failed: Identity DID Path Missing"; exit 1;} || echo "Identity DID Received";
-[ -z "$identity_name" ] && { echo "Script Failed: Identity DID Name Missing"; exit 1;} || echo "Identity Name Received";
+admin_did=$1
+admin_seed=$2
+identity_did=$3
+identity_seed=$4
```

Comment:
No identity_seed to be passed as the admin identity and new identity may come from different organisations. The job itself will run in the K8S cluster of the admin organisation thus just the private crypto of the admin might be used.